phoenix5-spark3/src/main/java/org/apache/phoenix/spark/sql/connector/reader/PhoenixDataSourceReadOptions.java

```diff
@@ -34,7 +34,7 @@ class PhoenixDataSourceReadOptions implements Serializable {
     PhoenixDataSourceReadOptions(String zkUrl, String scn, String tenantId,
                                  String selectStatement, Properties overriddenProps,
         byte[] pTableCacheBytes) {
-        if(overriddenProps == null){
+        if (overriddenProps == null) {
```

Comment:
Can you undo whitespace-only changes for this patch ?
It would make it easier the find the real changes.

Reply:
Sure, I removed the checkstyle related fixes, I will do it in a next round.